src/IECoreDelightModule/IECoreDelightModule.cpp

```diff
@@ -0,0 +1,44 @@
+//////////////////////////////////////////////////////////////////////////
+//
+//  Copyright (c) 2012, John Haddon. All rights reserved.
```

Comment:
2023?

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5054/commits/6f3e95575424087f2e20737b45df5731e5c0cd2f.